### PR TITLE
Yieldmo Bid Adapter : send topics in the Bid Request

### PR DESCRIPTION
**Pull request by desidiver**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Use the Topics module to pass topics in the bid request.

### CODE REVIEW

**Comment by desidiver** [comment]
@ChrisHuie Been waiting for this review for a while now. Thank you.

**Review by ChrisHuie — APPROVED** [review]

**Comment by ChrisHuie** [comment]
@desidiver looks fine to me. Can you throw some unit tests in here before i merge? 🙏

**Comment by desidiver** [comment]
> @desidiver looks fine to me. Can you throw some unit tests in here before i merge? 🙏

@ChrisHuie Added couple of unit tests. Let me know if that works. Thank you.

**Pull request merged**
